src/components/Reviews/SingleReviewPage.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import * as api from "../../api";
+import { ReactComponent as CommentIcon } from "../../icons/comment.svg";
+import { ReactComponent as VotesIcon } from "../../icons/votes.svg";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+dayjs.extend(relativeTime);
+
+const SingleReviewPage = () => {
+  const [isLoading, setLoading] = useState(true);
+  const [review, setReview] = useState([]);
+  const { review_id } = useParams();
+
+  useEffect(() => {
+    api.fetchSingleReview(review_id).then((review) => {
+      setReview(review);
+      setLoading(false);
+    });
+  }, []);
+
+  return (
+    <section className="h-max flex flex-col items-center columns-1 bg-stone-500 gap-5 mt-5">
```

Comment:
Section is fine here although article may be slightly more semmantic for screen readers